main.go

```diff
@@ -39,7 +39,7 @@ func main() {
 
 	http.Handle("/room", &signaling.Server{})
 
-	fs := http.FileServer(http.Dir("./static"))
+	fs := http.FileServer(http.Dir("./frontend/build"))
```

Comment:
Can we back this out? This will actually break out the ability to deploy the prototype.

Reply:
yeah I'm patching that up now I had swapped out static folders.